backends/cuda-reg/ceed-cuda-reg-restriction.c

```diff
@@ -211,6 +211,26 @@ static int CeedElemRestrictionApply_Cuda_reg(CeedElemRestriction r,
   return 0;
 }
 
+//------------------------------------------------------------------------------
+// Get offsets
+//------------------------------------------------------------------------------
+static int CeedElemRestrictionGetOffsets_Cuda_reg(CeedElemRestriction rstr,
+    CeedMemType mtype, const CeedInt **offsets) {
+  int ierr;
+  CeedElemRestriction_Cuda *impl;
+  ierr = CeedElemRestrictionGetData(rstr, (void *)&impl); CeedChk(ierr);
+
+  switch (memtype) {
+  case (CEED_MEM_HOST)
+      *offsets = impl->h_ind;
+    break;
+  case (CEED_MEM_DEVICE)
```

Comment:
Odd style; no parens needed in the case statement. Also, why does this need to be different from `CeedElemRestrictionGetOffsets_Cuda` (which looks like a copy).

Reply:
Typo there on my part. The two backends have different backend data structs.